receiver/scraperhelper/scrapercontroller.go

```diff
@@ -184,7 +188,14 @@ func (sc *controller) startScraping() {
 
 			sc.tickerCh = ticker.C
 		}
+		if sc.initialDelay > 0 {
```

Comment:
If I understand correctly, this means that if I configure CollectionInterval to 10s and InitialDelay to 2s, the scrape will occur at the following times since start of collector: 2s, 10s, 20s, 30s, ...

Shouldn't the initial delay also delay the following scrapes? I would assume the scrapes would occur at: 2s, 12s, 22s, 32s, ...

Reply:
Yes, you're example is correct. 

I think the ordering should be fixed up to address that to align with what I imagine most users would expect and make it clear in the change log.

Reply:
Updated :)

Reply:
Great, thanks!